release/README.md

```diff
@@ -7,7 +7,7 @@
   ```
 * Make sure you have uploaded your aws credentials to your user secrets.
   ```bash
-  ./earthly secrets ls /user/earthly-technologies/aws/credentials
+  ./earthly secrets get /user/earthly-technologies/aws/credentials
```

Comment:
This ensures it is not a false positive, and that there are valid creds in there.